mobile/lib/shared/models/asset.dart

```diff
@@ -116,6 +118,8 @@ class Asset {
   /// because Isar cannot sort lists of byte arrays
   String checksum;
 
+  String? thumbhash;
```

Comment:
Please make this a `List<byte>` to store the raw bytes instead of the longer base64. This also avoids the wasteful decoding of base64 to bytes on every render.

Reply:
Updated. Thank you 😄